snuba/query/processors/object_id_rate_limiter.py

```diff
@@ -188,7 +170,7 @@ def get_object_id(
         return str(obj_id)
 
 
-class ReferrerRateLimiterProcessor(OnlyIfConfiguredRateLimitProcessor):
+class ReferrerRateLimiterProcessor(ObjectIDRateLimiterProcessor):
```

Comment:
I don't think this should inherit from the `ObjectIDRateLimiterProcessor`. It isn't filtering by an object ID at all. It can be its own separate class.